src/library/scala/collection/Parallelizable.scala

```diff
@@ -43,6 +46,31 @@ trait Parallelizable[+A, +ParRepr <: Parallel] extends Any {
     cb.result()
   }
 
+  /** Returns a parallel implementation of this collection using the given thread pool.
+   *
+   * @param pool the thread pool to use for supporting parallelism
+   * @return a parallel implementation of this collection using `pool` for its threading
+   */
+  final def parWith(pool: ForkJoinPool): ParRepr = parWith(new ForkJoinTaskSupport(pool))
+
+  /** Returns a parallel implementation of this collection using the given task support.
+   *
+   * @param ts the task support to use for supporting parallelism
+   * @return a parallel implementation of this collection using `ts` for parallelism
+   */
+  final def parWith(ts: TaskSupport): ParRepr = {
+    val pseq = par
+    setTaskSupport(pseq, ts)
+    pseq
+  }
+
+  /** Returns a parallel implementation of this collection using the given number of threads.
+   *
+   * @param pool the number of threads to use for supporting parallelism
+   * @return a parallel implementation of this collection using `n` threads
+   */
+  final def parWith(n: Int): ParRepr = parWith(new ForkJoinPool(n))
```

Comment:
I suppose I could live without it, although I like the `Int` overloading because it allows me to simply say "just give me n-fold parallelism", and not have to muck about with all the `ForkJoinPool` or `ForkJoinTaskSupport`.

It may be that the primary use case is experimenting with throughput as a function of number of threads, but it's a thing I do with some frequency.